### PR TITLE
Fix the parameter name of deliveryserviceserver in document

### PR DESCRIPTION
**Pull request by Shihta**
## Which issue is fixed by this PR? If not related to an existing issue, what does this PR do?

simply fix the parameter name

## Which TC components are affected by this PR?

- [x] Documentation
- [ ] Grove
- [ ] Traffic Analytics
- [ ] Traffic Monitor
- [ ] Traffic Ops
- [ ] Traffic Ops ORT
- [ ] Traffic Portal
- [ ] Traffic Router
- [ ] Traffic Stats
- [ ] Traffic Vault
- [ ] Other _________

### (If no tests are part of this PR, please provide explanation as to why no tests are included.)

1. Manually confirm that the name I modified is correct
2. Using `./pkg -v docs` to verify the document can be built correctly

## Check all that apply

- [ ] This PR includes tests
- [x] This PR includes documentation updates
- [ ] This PR includes an update to CHANGELOG.md
- [ ] This PR includes all required license headers
- [ ] This PR includes a database migration (ensure that migration sequence is correct)
- [ ] This PR fixes a serious security flaw. Read more: [www.apache.org/security](http://www.apache.org/security/)

__

### CODE REVIEW

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/trafficcontrol-PR/3442/
Test PASSed.

File: docs/source/api/deliveryserviceserver.rst [hunk 1]
```diff
@@ -107,9 +107,9 @@ Assign a set of one or more servers to a :term:`Delivery Service`
 
 Request Structure
 -----------------
-:deliveryService: The integral, unique identifier of the :term:`Delivery Service` to which the servers identified in the ``servers`` array will be assigned
-:replace:         If ``true``, any existing assignments for a server identified in the ``servers`` array will be overwritten by this request
-:servers:         An array of integral, unique identifiers for servers which are to be assigned to the :term:`Delivery Service` identified by ``deliveryService``
+:dsId:    The integral, unique identifier of the :term:`Delivery Service` id to which the servers identified in the ``servers`` array will be assigned
```
Comment (ocket8888) [inline docs/source/api/deliveryserviceserver.rst]: we don't need to specify twice that this is an identifier, "integral, unique identifier" already does that, so adding "id" is redundant.

Comment (Shihta) [reply docs/source/api/deliveryserviceserver.rst]: fixed

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/trafficcontrol-PR/3494/
Test FAILed.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/trafficcontrol-PR/3496/
Test PASSed.

**Pull request merged**